libs/blocks/notification/notification.js

```diff
@@ -89,14 +89,76 @@ function wrapCopy(foreground) {
 function addCloseAction(el, btn) {
   btn.addEventListener('click', (e) => {
     if (btn.nodeName === 'A') e.preventDefault();
+
+    // Create accessibility live region
```

Comment:
can probably remove this comment, we try to keep as many as we can out and the variable name is good